src/App.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
+import {MovieList} from './components/MovieList';
+import {MediaPlayer} from './components/MediaPlayer';
```

Comment:
**Context:**
stick to the convention of adding spaces before and after the named import curly brackets.
airbnb style guides is a standard and it is followed almost everywhere and you can see that they have the spaces before and after the named imports. furthermore, prettier (which is an automated styleguide enforcement tool, build by a facebook developer and React is using this as well) is vastly used in react ecosystem, also include spaces before and after named imports.

**Action Item:** Make sure that all named imports in all commit files have spaces before and after the named import in curly brackets like this { Component }

https://github.com/airbnb/javascript#modules--no-duplicate-imports
https://github.com/prettier/prettier